localstack/services/s3/s3_utils.py

```diff
@@ -302,14 +302,15 @@ def authenticate_presign_url_signv2(method, path, headers, data, url, query_para
             message='The request signature we calculated does not match the signature you provided. \
                     Check your key and signing method.')
 
-    # Checking whether the url is expired or not
-    if int(query_params['Expires'][0]) < time.time():
-        return requests_error_response_xml_signature_calculation(
-            code=403,
-            code_string='AccessDenied',
-            message='Request has expired',
-            expires=query_params['Expires'][0]
-        )
+    if config.S3_SKIP_SIGNATURE_VALIDATION:
```

Comment:
Thanks for this PR @macnev2013 . In the current version, we'd be only returning the error if `S3_SKIP_SIGNATURE_VALIDATION` is set (but the logic should be the other way around).

I think we need something similar to this here:
```
    if int(query_params['Expires'][0]) < time.time():
        if config.S3_SKIP_SIGNATURE_VALIDATION:
            LOGGER.warning('Signature is expired, but not raising an error, as S3_SKIP_SIGNATURE_VALIDATION=1')
        else:
            return requests_error_response_xml_signature_calculation(...)
```